src/recipients/RecipientItemV2.tsx

```diff
@@ -34,12 +34,17 @@ function RecipientItem({ recipient, onSelectRecipient, loading, selected }: Prop
   }
 
   const e164NumberToAddress = useSelector(e164NumberToAddressSelector)
+  const verifiedAddresses = useSelector(verifiedAddressesSelector)
 
   // TODO(ACT-980): avoid icon flash when a known valora contact is clicked
-  // TODO(ACT-950): show icon for address recipients
   const showValoraIcon = useMemo(() => {
     if (recipient.recipientType === RecipientType.PhoneNumber) {
       return recipient.e164PhoneNumber && !!e164NumberToAddress[recipient.e164PhoneNumber]
+    } else if (
+      recipient.recipientType === RecipientType.Address ||
+      recipient.recipientType === RecipientType.Nomspace
```

Comment:
we can probably drop these conditions? The only other recipient type is Merchant, which I am not even sure if its used anymore. I see 0 references to it.

Reply:
Yeah, agreed -- dropped this if statement.